src/Pages/Main/Components/MoviePosters/MoviePosters.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from "react";
+import MoviePosterItem from "../MoviePosterItem/MoviePosterItem";
+import "./MoviePosters.scss";
+
+export default class MoviePosters extends Component {
+  constructor() {
+    super();
+    this.state = {
+      data: [],
+    };
+  }
+
+  componentDidMount = () => {
+    fetch("http://localhost:3000/data/MainPageMockData/MainpageMockData.json")
+      .then((res) => res.json())
+      .then((res) => {
+        this.setState({
+          data: res.data,
+        });
+      });
+  };
+  render() {
+    return (
+      <div className="MoviePosters">
+        <ul className="moviePoster">
+          {this.state.data.map(
+            ({
+              id,
+              korean_title,
+              poster_url,
+              year,
+              country,
+              avg_score,
+              service_provider,
+            }) => {
+              return (
+                <MoviePosterItem
+                  key={id}
+                  korean_title={korean_title}
+                  poster_url={poster_url}
+                  year={year}
+                  country={country[0]}
+                  avg_score={avg_score}
+                  service_provider={service_provider}
+                />
+              );
+            }
+          )}
```

Comment:
map 함수 안에서 객체 비구조화 할당 너무 잘 사용해주셨네요! 하지만 저렇게 필드가 많은경우 그리고 한번씩만 사용하는 경우 저렇게 비구조화 할당을 함으로서 코드 가독성이 크게 떨어질 수 있습니다~ 아래와 같이 변경해주신다면 조금 더 한눈에 들어오는 코드가 될 수 있겠네요 😊

비구조화 할당은 state 와 props 에 활용해주시는 편이 조금 더 효율적일 것 같습니다!

```suggestion
          {this.state.data.map(el => (
            <MoviePosterItem
              key={el.id}
              korean_title={el.korean_title}
              poster_url={el.poster_url}
              year={el.year}
              country={el.country[0]}
              avg_score={el.avg_score}
              service_provider={el.service_provider}
            />
          ))}
```

Reply:
넵 확실히 간결해졌네요! 적용했습니다 감사합니다!